test/clt-tests/core/test-unfreeze-defrosts.rec

```diff
@@ -62,3 +62,18 @@ sleep 5; mysql -h0 -P9306 -e "select count(*) from orders\G"
 ––– output –––
 *************************** 1. row ***************************
 count(*): 10000
+––– input –––
+mysql -h0 -P9306 -e "drop table if exists t; create table t; freeze t; flush ramchunk t;" & sleep 1; true
+––– output –––
+[2] %{NUMBER}
++---------------------------------+---------------------------------+
+| file                            | normalized                      |
++---------------------------------+---------------------------------+
+| /var/lib/manticore/t/t.meta     | /var/lib/manticore/t/t.meta     |
+| /var/lib/manticore/t/t.ram      | /var/lib/manticore/t/t.ram      |
+| /var/lib/manticore/t/t.settings | /var/lib/manticore/t/t.settings |
++---------------------------------+---------------------------------+
+––– input –––
+searchd --stop > /dev/null 2>&1; if timeout 60 grep -qm1 'TERM' <(tail -n 1000 -f /var/log/manticore/searchd.log); then echo 'Searchd stopped.'; else echo 'Timeout or failed!'; cat /var/log/manticore/searchd.log; fi
```

Comment:
I'm not sure `grep TERM` is enough to test that the searchd stopped successfully. There should be smth like 'Shutdown complete'